src/plugins/session.js

```diff
@@ -135,11 +139,10 @@ module.exports.register = function(server, options, next) {
           '</script>' +
           '</head>' +
           '</html>');
```

Comment:
Could we `.unstate(nonceKey).unstate(stateKey)` here?

Reply:
i tested using `unstate`, and that didn't seem to matter

Reply:
I think I misunderstood this comment. adding `unstate` where you suggest will make difference. I'll update similar to https://github.com/auth0-extensions/auth0-extension-express-tools/pull/16 and create another PR.